pkg/apis/psmdb/v1/psmdb_defaults.go

```diff
@@ -199,6 +201,10 @@ func (cr *PerconaServerMongoDB) CheckNSetDefaults(platform version.Platform, log
 			replset.ReadinessProbe.FailureThreshold = int32(8)
 		}
 
+		if len(replset.ServiceAccountName) == 0 {
```

Comment:
it should be added only in version >= 1.5.0

Reply:
ok